frontend/src/components/CancelTransferBtn.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import {useContractWrite} from "@starknet-react/core";
+import {TREASURY_ADDRESS} from "../lib/config";
+import {Contract} from "starknet";
+import TreasuryABI from "../lib/treasury_abi.json";
+
+interface CancelTransferButtonProps {
+    transferId: number;
+}
+
+const CancelTransferBtn: React.FC<CancelTransferButtonProps> = ({transferId}) => {
+
+    const contract = new Contract(TreasuryABI, TREASURY_ADDRESS)
```

Comment:
Here, we use `treasury_abi.json`, but in StatusTransfer.tsx, treasuryABI.ts is used. This should be unified